src/subresource-signing.js

```diff
@@ -39,8 +54,10 @@ var srs = {
         /**
          * Assume the contents is valid, and attach the script
          * to the document so it runs.
+         *
+         * @param contents
          */
-        attach_script: function srs_attach_script(contents) {
+        attach_script: function(contents) {
```

Comment:
By the way, this was new to me too (learning something). See here for more info: https://kangax.github.io/nfe/#named-expr

I never see this and maybe this is why? When I tried this in Chrome, I couldn't make that available outside of the scope of the function itself. Could have been a Chrome thing but turns out maybe it's spec (didn't see a citation on that claim, however). Mozilla tends to be pretty good at documenting stuff, see here as well: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function#Named_function_expression